libs/api/domains/user-profile/src/lib/userProfile.service.ts

```diff
@@ -84,26 +81,84 @@ export class UserProfileService {
       nationalId: user.nationalId,
       //temporary as schemas where not working properly
       locale: input.locale as string,
-      mobilePhoneNumber: input.mobilePhoneNumber,
-      email: input.email,
+
+      /* Mobile and email will be within islykill service */
+      // mobilePhoneNumber: input.mobilePhoneNumber,
+      // email: input.email,
     }
     const request: UserProfileControllerCreateRequest = {
       createUserProfileDto: createUserDto,
     }
-    return await this.userProfileApiWithAuth(user)
+
+    const feature = await this.featureFlagService.getValue(
+      Features.personalInformation,
+      false,
+      user,
+    )
+
+    if (feature) {
+      await this.islyklarService
+        .createIslykillSettings(user.nationalId, {
+          email: input.email,
+          mobile: input.mobilePhoneNumber,
+          canNudge: input.canNudge,
+        }) // Current version does not return the newly created user in the response.
+        .catch(handleError)
+    } else {
+      logger.error('User profile create is feature flagged for user')
```

Comment:
Should be info/warning to not raise alarms in production if someone changes their locale. Same in the other mutation.

Reply:
Yes! That makes sense. 🙈